scripts/language_model/transformer/transformer.py

```diff
@@ -27,7 +27,7 @@
 import gluonnlp as nlp
```

Comment:
Let's not change the permissions of this file from `644` to `755`. Please run `chmod ugo-x transformer.py` to remove execute permission.